index.html

```diff
@@ -2111,6 +2119,8 @@ <h2>Sessions</h2>
 A <a>session</a> has an associated <a>user prompt handler</a>.
 Unless stated otherwise it is in the <a>dismiss and notify state</a>.
 
+<p>A <a>session</a> has an associated <a>executable script map</a>.
```

Comment:
This needs to be something like:

> A _session_ has an associated _map_ of _executable scripts_.

Then you need to add [map](https://infra.spec.whatwg.org/#maps) to the [Index](https://w3c.github.io/webdriver/#index) under the _Infra_ entry.